### PR TITLE
[fontconfig] fixes for arm64-macos

### PR DESCRIPTION
**Pull request by abique**
References:
 - https://github.com/Homebrew/homebrew-core/pull/57999
 - https://github.com/Homebrew/homebrew-core/issues/57994

### CODE REVIEW

File: ports/fontconfig/macos_arm_fccache.patch [hunk 1]
```diff
@@ -0,0 +1,43 @@
+From 6def66164a36eed968aae872d76acfac3173d44a Mon Sep 17 00:00:00 2001
+From: Nirbheek Chauhan <nirbheek@centricular.com>
+Date: Sat, 22 Feb 2020 03:57:50 +0530
+Subject: [PATCH] fcatomic: Fix EXC_BAD_ACCESS on iOS ARM64
+
+iPhone 2.1 was released a long time ago, and the macro for checking
+the target iOS SDK version has changed. We can simplify everything and
+do a very basic check.
+---
```
Comment (PhoebeHui) [inline ports/fontconfig/macos_arm_fccache.patch:9]:
```suggestion
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@abique, thanks for the PR! 

Could you minimize the patch?

File: ports/fontconfig/macos_arm_fccache.patch (hunk 1 above)
Comment (abique) [reply ports/fontconfig/macos_arm_fccache.patch:9]: Hi @PhoebeHui,

Thank you for reviewing the change.

Why would we minimize it?
I think it provides context which can help if there is a problem with this patch in the future.

Through if you insist, I don't mind applying your changes.

Regards,
Alex

Comment (abique) [reply ports/fontconfig/macos_arm_fccache.patch:9]: Hi @PhoebeHui,

Thank you for reviewing the change.

Why would we minimize it?
I think it provides context which can help if there is a problem with this patch in the future.

Through if you insist, I don't mind applying your changes.

Regards,
Alex

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

**Comment by abique** [comment]
Thank you for the review :)

**Pull request merged**